projects/hslayers/src/components/compositions/endpoints/compositions-status-manager.service.ts

```diff
@@ -50,9 +52,19 @@ export class HsCompositionsStatusManagerService {
       ds.listLoading.unsubscribe();
       delete ds.listLoading;
     }
-    ds.listLoading = this.$http.get(url).subscribe(
+    const listLoading = this.$http.get(url).pipe(
+      timeout(2000),
```

Comment:
Why is timeout necessary?

Reply:
To timeout failed or pending requests, that are slowing down the composition loading.